src/main/java/io/github/bonigarcia/wdm/Downloader.java

```diff
@@ -52,6 +55,27 @@
 
 	private static final String HOME = "~";
 
+
+	public static Proxy createProxy() {
+		String proxyString = System.getenv("HTTPS_PROXY");
+		if (proxyString == null || proxyString.length() < 1)
```

Comment:
Repeated condition. In addition, please use always { and }, even if the block is a single line.